source/_posts/2021-04-07-release-20214.markdown

```diff
@@ -137,6 +137,12 @@ Furthermore two new services are introduced:
 - `zwave_js.bulk_set_partial_config_parameters`
 - `zwave_js.set_value`
 
+<div class='note'>
+There are breaking changes in the Z-Wave JS integration that could affect
+your existing automations. Be sure to read the
+[breaking changes](#breaking-changes) section for more info.
+</div>
```

Comment:
We don't use the notes in release notes, as they are really intrusive.

```suggestion
There are breaking changes in the Z-Wave JS integration that could affect your existing automations. Be sure to read the [breaking changes](#breaking-changes) section for more info.
```

I will ensure this is mentioned high up this paragraph for the final notes.

Reply:
Sounds good, appreciate it